sample/kotlin/src/main/kotlin/com/datadog/android/sample/server/LocalServer.kt

```diff
@@ -37,7 +37,9 @@ class LocalServer {
     }
 
     fun stop() {
-        engine?.stop(SHUTDOWN_MS, STOP_TIMEOUT_MS, TimeUnit.MILLISECONDS)
+        Thread {
```

Comment:
Any specific reason ?

Reply:
Yes, the stop method in the server will block the main thread for about a second before returning.

Reply:
In this case shouldn't be the case also with the start method ? I am thinking maybe you should also start the server in another Thread. Problem is that doing so you will have to synchronise your code to wait until the server actually starts.

Reply:
No cause the start is asynchronous. To be more accurate, stop is not synchronous, it just sends a "stop" signal to the server and waits at least SHUTDOWN_MS and at most STOP_TIMEOUT_MS, effectively blocking the calling thread. 
And no, we don't need to synchronise the code for start/stop, the library handles it safely